eclipse-theia/theia-blueprint
# Theia IDE Rebrand and Theia 1.45.0

#### What it does
* Rebrand Theia Blueprint as Theia IDE 
* Update to Theia 1.45.0
* Fix Javascript Debugging

Fixes #318
Fixes #317

Contributed on behalf of STMicroelectronics

#### How to test
* build the application and check everything is still working
* Check welcome page, etc to see if there are places where the Blueprint->IDE rename is missing

#### Review checklist


#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
